src/test/resources/fixtures/ifs/if-avoid-not-operator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+if (<weak_warning descr="This negative if conditional could be avoided">!expr()</weak_warning>) { a(); b(); }
+else { c(); d(); }
+
+if (!true) { a(); b(); }
```

Comment:
This is partially ignored. On  reality, the if() itself is ignored, then the last elseif() branch is inspectioned. Futurely I hope could treats complex cases like that, but for now it will works fine. Have you some code in mind where it will be a problem?